samples/tls/logging/README.md

```diff
@@ -22,39 +39,7 @@ my_repository_path/logging/bin/deploy_logging_open.sh
 ```
 ## Notes On Customization Values
 
-### TLS 
-
 Specify `TLS_ENABLE=true` in the `user.env` file to require TLS for connections to Kibana. Connections to Elasticsearch (when enabled) always use TLS, regardless of the value of `TLS_ENABLE`.
 
-### Ingress
-
-* If you plan on using ingress, you must manually populate these Kubernetes secrets with TLS certificates before you run the deployment script:
-
-* `kibana-ingress-tls-secret`
-* `elasticsearch-ingress-tls-secret`
-
-After you have obtained the certificates for the secrets, use this command to generate the secrets:
-
-```bash
-kubectl create secret tls $SECRET_NAME -n $NAMESPACE --key $CERT_KEY --cert $CERT_FILE
-```
-
-Use `kibana-ingress-tls-secret` and `elasticsearch-ingress-tls-secret` as values for `$SECRET_NAME`. Use `logging` for the value of `$NAMESPACE`.
-
-The process of generating the certificates for these secrets is out of scope for this example.
-
-* If you are using an ingress controller other than NGINX, modify the annotation 
-`nginx.ingress.kubernetes.io/backend-protocol: HTTPS` as needed in the `user-values-elasticsearch-open.yaml` file. Refer to the documentation for your ingress controller. 
-
```

Comment:
I think the Ingress section needs to be kept.  It is NOT covered in the NOTES_ON_TLS.md doc.